server/src/main/java/org/opensearch/search/searchafter/SearchAfterBuilder.java

```diff
@@ -201,9 +201,15 @@ private static Object convertValueFromSortType(String fieldName, SortField.Type
 
                 case LONG:
                     // for unsigned_long field type we want to pass search_after value through formatting
-                    if (value instanceof Number && format != DocValueFormat.UNSIGNED_LONG_SHIFTED) {
```

Comment:
Just curious , what was the reason we didnt support search_after earlier for unsigned_long and now we can ?

Reply:
The `unsigned_long` (in some form) was supported when `DocValueFormat.UNSIGNED_LONG_SHIFTED` was used, but I to be fair do not know how this value was used later on (this is/was x-pack feature, I have not looked into the code).